script/configure-integration-tests.ps1

```diff
@@ -43,21 +43,34 @@ function VerifyEnvironmentVariable([string]$friendlyName, [string]$key, [bool]$o
     else
     {
        Write-Host "$existing_value found as the configured $friendlyName"
-       $reset = Read-Host -Prompt "Want to change this? Press Y, otherwise we'll move on"
-       if ($reset -eq "Y")
-       {
-           $value = Read-Host -Prompt "Change the $friendlyName to use for the integration tests"
-           SetVariable $key $value
-       }
-
        if ($optional -eq $true)
        {
-            $clear = Read-Host -Prompt 'Want to remove this optional value, press Y'
+	     $clear = Read-Host -Prompt 'Want to remove or change this optional value, press Y'
             if ($clear -eq "Y")
             {
-                SetVariable $key $null
-            }
-       }
+                $reset = Read-Host -Prompt "Press R to remove and Press C to change the value, otherwise we'll move on"
+		          if ($reset -eq "C")
+         		 {
+               		$value = Read-Host -Prompt "Change the $friendlyName to use for the integration tests"
+       		        SetVariable $key $value
+		        }
+	        	elseif ($reset -eq "R")
+		        { 
+        			SetVariable $key $null
+	        	}
+             }
+            
+        }
+	   else
+	   {
+	    $reset = Read-Host -Prompt "Want to change this value , press Y, otherwise we'll move on"
```

Comment:
Let's tweak this message: `Press Y to change this value, otherwise we'll move on"